lib/core/controlplane.rb

```diff
@@ -143,40 +143,38 @@ def query_workloads(workload, a_gvc = gvc, a_org = org, partial_workload_match:
     api.query_workloads(org: a_org, gvc: a_gvc, workload: workload, gvc_op_type: gvc_op, workload_op_type: workload_op)
   end
 
-  def workload_get_replicas(workload, location:)
-    cmd = "cpln workload get-replicas #{workload} #{gvc_org} --location #{location} -o yaml"
+  def fetch_workload_replicas(workload, location:)
+    cmd = "cpln workload replica get #{workload} #{gvc_org} --location #{location} -o yaml"
```

Comment:
`replica get` is same as `get-replicas` (the latter was deprecated for the former).